packages/react-native/ReactCommon/react/renderer/graphics/Transform.h

```diff
@@ -50,6 +51,23 @@ struct TransformOperation {
   Float z;
 };
 
+struct TransformOrigin {
+  std::array<YGValue, 2> xy;
```

Comment:
Per @NickGerleman feedback on another PR, using YGValue is an undesired dependency. Let's introduce a custom struct for this in another PR.

Reply:
Added a PR for struct - https://github.com/facebook/react-native/pull/39150. I have tested it replacing `YGValue` and it works, kept it simple. let me know!

`std::array<YGValue, 2> xy;` -> `std::array<ValueUnit, 2> xy;`  

cc - @jacobp100 we can also use it here - https://github.com/facebook/react-native/pull/38626/

Reply:
Will try and get to it later this weekend